docs/design.md

```diff
@@ -127,7 +125,6 @@ The example Model's backing storage would end up being:
 
 ```sh
 gs://abc123-substratus-artifacts/f94a0d128bcbd9c1b824e9e5572baf86/model/
```

Comment:
I think there is no longer a good reason to store model in a subdirectory

Reply:
agreed, I think this doc is wrong now, will fix